package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please add the `octokit` dependency used in the GitHub action as a package dependency.

Reply:
added. don't know why it was working for my project without it.